### PR TITLE
[stable/postgresql] Unify and document production values

### PR DESCRIPTION
**Pull request by carrodher**
### What this PR does / why we need it:

Unify and document production values

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [X] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [X] Chart Version bumped
- [X] Variables are documented in the README.md
- [X] Title of the PR starts with chart name (e.g. `[stable/chart]`)

### CODE REVIEW

File: stable/postgresql/README.md [hunk 1]
```diff
@@ -187,6 +187,51 @@ $ helm install --name my-release -f values.yaml stable/postgresql
...
 
+### Production configuration
+
+This chart includes a `values-production.yaml` file where you can find some parameters oriented to production configuration in comparison to the regular `values.yaml`.
+
+```console
+$ helm install --name my-release -f ./values-production.yaml stable/postgresql
+```
+
+- Would you like to enable replication:
```
Comment (javsalgar) [inline stable/postgresql/README.md]: Enable replication

**Comment by javsalgar** [comment]
Small change

**Review by tompizmor — APPROVED** [review]
/ok-to-test
/lgtm

**Comment by tompizmor** [comment]
@carrodher can you bump the version again? There has been already a release for Postgresql `5.3.6`

**Review by tompizmor — APPROVED** [review]
/ok-to-test
/lgtm

**Pull request merged**